core/lib/types/src/l2_to_l1_log.rs

```diff
@@ -63,6 +55,22 @@ impl L2ToL1Log {
     }
 }
 
+/// Returns the minimum number of items in the Merkle tree built from L2-to-L1 logs
+/// for a certain protocol version.
+pub fn min_l2_to_l1_logs_tree_size(protocol_version: ProtocolVersionId) -> usize {
```

Comment:
this wont work if we're using 1.4.2 with 1 blob since we'll only be able to handle 2048 logs

Reply:
Why? This function sets a height for a tree. Pubdata seal criterion (which has to do with this function) would close batches by setting pubdata limit depending on config

Reply:
doesn't it affect the size of the mini merkle tree thats constructed? which would change the root [here](https://github.com/matter-labs/zksync-era/blob/80c50291c8f8439b1a9c11d680ebd00c85959e52/core/lib/types/src/commitment/mod.rs#L302)

Reply:
Yes, it affects. But I still don't get why does it matter

Reply:
just wanted to double check that we didn't end up with more logs than we could fit in a batch. it sound like this is a non issue though so ill resolve